content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -493,6 +493,18 @@ you through the steps you follow to apply a seccomp profile to a Pod or to one o
 its containers. That tutorial covers the supported mechanism for configuring seccomp in Kubernetes,
 based on setting `securityContext` within the Pod's `.spec`.
 
+### snapshot.storage.kubernetes.io/allowVolumeModeChange
+
+Example: `snapshot.storage.kubernetes.io/allowVolumeModeChange: true`
```

Comment:
I'd have this as:
```suggestion
Example: `snapshot.storage.kubernetes.io/allowVolumeModeChange: "true"`
```

Reply:
@RaunakShah : PTAL since this hasn't been addressed and we believe it improves readability of the doc.

Reply:
Done